src/Main.java

```diff
@@ -1,5 +1,126 @@
+import java.text.NumberFormat;
+import java.util.Random;
+import java.util.Scanner;
+
 public class Main {
+    private final static Employee[] employees = new Employee[10];
```

Comment:
```suggestion
    private final static Employee[] EMPLOYEES = new Employee[10];
```
т.к. static и final одновременно